projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -43,21 +43,19 @@ const ActionButton = ( {
 	status,
 	admin,
 	name,
-	onLearn,
 	onActivate,
-	onAdd,
 	onManage,
 	onFixConnection,
 	isFetching,
 } ) => {
 	if ( ! admin ) {
 		return (
-			<Button variant="link" onClick={ onLearn } className={ styles[ 'action-link-button' ] }>
+			<span className={ styles[ 'action-link-button' ] }>
```

Comment:
Instead of moving out of `Button` you could use this prop: https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/button/index.js#L87 ... Since we already are using `__experimental` stuff at My Jetpack.

Reply:
not sure how it could fit with what we need to achieve. Basically, we need to make the whole Product Card element focusable, and remove it from from the button.

Reply:
@retrofox this fit to avoid you moving out of the `Button` component and just disable the `focus` action, that highlights the entire button, that now happens on the entire card

Reply:
ok, but the intention here is making the whole card clickable, right?

<img width="720" alt="image" src="https://user-images.githubusercontent.com/77539/159264401-c04d0823-ace2-45fa-9176-d8b0b0cc7f2c.png">

Why are we going to implement also the text (Add Backup in the image above) clickable when it's entirely handled by the main wrap element?

Reply:
That makes sense, let's move forward this way 🙃